core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/SubmissionCostService.scala

```diff
@@ -1,14 +1,14 @@
 package org.broadinstitute.dsde.rawls.dataaccess
 
-import java.util
-
-import org.broadinstitute.dsde.workbench.google.GoogleBigQueryDAO
-import com.google.api.services.bigquery.model.{GetQueryResultsResponse, QueryParameter, QueryParameterType, QueryParameterValue, TableRow}
+import com.google.api.services.bigquery.model._
```

Comment:
Do we have an established preference between _ imports and specific imports?

Reply:
I just let intellij organize the imports so shrug